README.md

```diff
@@ -1,43 +1,32 @@
 # Elasticsearch Operator
 
-This is a simple operator for running Elasticsearch in Kubernetes. It's an
-alternative to the
-[upmc-enterprises/elasticsearch-operator](https://github.com/upmc-enterprises/elasticsearch-operator)
-with more focus on operational aspects, like draining of Elasticsearch data nodes, rather than
-abstracting manifest definitions. One key feature is upgrading of
-Elasticsearch data nodes, as well has safely handling cluster upgrades while being
-highly available.
-
-The other key feature is about auto-scaling of Elasticsearch data nodes.
+This is an operator for running Elasticsearch in Kubernetes with focus on operational aspects, like safe draining of Elasticsearch data nodes, rather than just abstracting manifest definitions.
 
+One key feature is upgrading of Elasticsearch data nodes, as well as safely handling cluster upgrades while being highly available. The other key feature is offering auto-scaling for Elasticsearch data nodes.
```

Comment:
Hmm maybe we can drop this? We already have key features below. So either add this to key features or completely drop it, WDYT?

Reply:
True, it's a bit confusing to talk about key features here, and below. I'll condense this into a one-sentence sales pitch. WDYT about this: 
```
This is an operator for running Elasticsearch in Kubernetes with focus on operational aspects, like safe draining and offering auto-scaling capabilities for Elasticsearch data nodes, rather than just abstracting manifest definitions.
```